src/site/content/en/blog/how-to-file-a-good-bug/index.md

```diff
@@ -152,20 +152,20 @@ What steps will reproduce the problem?
 3. Click Install on the browser modal install confirmation.
 ```
 
-And finally, describe the *actual*, and *expected* result.
+And finally, describe the *expected*, and *actual* result.
 
 ```text
-What is the actual result? In the console:
-0. INSTALL: Available (logged when `beforeinstallprompt` event fired)
-1. INSTALL: Success (logged when `appinstalled` event fired)
-2. INSTALL_PROMPT_RESPONSE: {outcome: "accepted", platform: "web"}
-   (logged when beforeinstallprompt.prompt()` resolves)
-
 What is the expected result? In the console:
 0. INSTALL: Available (logged when `beforeinstallprompt` event fired)
 1. INSTALL_PROMPT_RESPONSE: {outcome: "accepted", platform: "web"}
    (logged when beforeinstallprompt.prompt()` resolves)
 2. INSTALL: Success (logged when `appinstalled` event fired)
+
+What is the actual result? In the console:
+0. INSTALL: Available (logged when `beforeinstallprompt` event fired)
+1. INSTALL: Success (logged when `appinstalled` event fired)
+2. INSTALL_PROMPT_RESPONSE: {outcome: "accepted", platform: "web"}
+   (logged when beforeinstallprompt.prompt()` resolves)
```

Comment:
I don't understand what I'm looking at, so I don't know how to review it. Is this a procedure? A list of possible outputs from something?